### ISSUE TITLE
Option file of the model trained on the DF2K dataset

### ISSUE THREAD

**Issue by liucencen54** [YuzhenD/Resyn#2]
Here is my option file (the yml file):
# general settings for image SR training
name: RWKVIRv4_SR_x4
model_type: RWKVIRModel
scale: 4
num_gpu: 8
manual_seed: 3407

# dataset and data loader settings
datasets:
  train:
    task: SR
    name: DF2K
    type: PairedImageDataset
    dataroot_gt:
      - /home/nas/datasets/DF2K/HR/
    dataroot_lq:
      - /home/nas/datasets/DF2K/LR_bicubic/X4/
    filename_tmpl: '{}x4'
    io_backend:
      type: disk

    gt_size: 128
    use_hflip: true
    use_rot: true

    # data loader
    use_shuffle: true
    num_worker_per_gpu: 4
    batch_size_per_gpu: 8
    dataset_enlarge_ratio: 100
    prefetch_mode: ~

  val:
    name: Urban100
    type: PairedImageDataset
    dataroot_gt: /home/nas/datasets/benchmark/Urban100/HR/
    dataroot_lq: /home/nas/datasets/benchmark/Urban100/LR_bicubic/X4/
    filename_tmpl: '{}x4'
    io_backend:
      type: disk

# network structures
network_g:
  type: RWKVIRv4
  upscale: 4
  in_chans: 3
  img_size: 64
  img_range: 1.
  depths: [6, 6, 6, 6, 6, 6]
  patch_size: 16
  embed_dim: 192
  mlp_ratio: 2
  channel_gamma: 0.25
  shift_pixel: 1
  hidden_rate: 4
  init_mode: 'fancy'
  upsampler: 'pixelshuffle'
  resi_connection: '1conv'

# path
path:
  pretrain_network_g: ~
  strict_load_g: true
  resume_state: ~

# training settings
train:
  optim_g:
    type: Adam
    #lr: !!float 2e-4
    lr: !!float 2e-4
    weight_decay: 0
    betas: [0.9, 0.99]

  scheduler:
    type: MultiStepLR
    milestones: [50000, 75000, 90000, 95000]
    #milestones: [5000, 10000, 15000, 20000，25000，30000，35000]
    gamma: 0.5

  total_iter: 500000
  warmup_iter: -1  # no warm up

  # losses
  pixel_opt:
    type: L1Loss
    loss_weight: 1.0
    reduction: mean
# validation settings
val:
  val_freq: !!float 5e3
  save_img: false

  metrics:
    psnr: # metric name, can be arbitrary
      type: calculate_psnr
      crop_border: 2
      test_y_channel: true

# logging settings
logger:
  print_freq: 200
  save_checkpoint_freq: !!float 5e3
  use_tb_logger: true
  wandb:
    project: ~
    resume_id: ~

# dist training settings
dist_params:
  backend: nccl
  port: 29500

**but the result is not same as the mentioned in paper, can you provide the yaml file for your training?**
![Uploading 1736169498204.jpg…]()

**Comment by YuzhenD** [comment]
You can try increasing the learning rate to 3e-4 or 4e-4.
